Source/Core/Common/FileUtil.cpp

```diff
@@ -59,6 +63,19 @@ namespace File
 static std::string s_android_sys_directory;
 #endif
 
+#ifdef __APPLE__
+static Common::DynamicLibrary s_security_framework;
+
+typedef Boolean (*dolSecTranslocateIsTranslocatedURL)(CFURLRef path, bool* isTranslocated,
```

Comment:
```suggestion
using DolSecTranslocateIsTranslocatedURL = Boolean (*)(CFURLRef path, bool* isTranslocated,
```

Ditto for the other typedef